src/EA.Weee.Web/Areas/AATFReturn/Controllers/NonObligatedController.cs

```diff
@@ -63,5 +73,20 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
             breadcrumb.ExternalActivity = activity;
             breadcrumb.SchemeInfo = await cache.FetchSchemePublicInfo(organisationId);
         }
+
+        private async void ValidateResult(NonObligatedValuesViewModel model, IWeeeClient client)
+        {
+            ReturnData @return;
+            @return = await client.SendAsync(User.GetAccessToken(), new GetReturn(model.ReturnId));
```

Comment:
Remove the ReturnData @return;
Use var @return = await client.SendAsync(User.GetAccessToken(), new GetReturn(model.ReturnId));